default.nix

```diff
@@ -0,0 +1,36 @@
+with (import ./.nix/nixpkgs.nix {});
+
+stdenv.mkDerivation rec {
+  pname = "haskell-report";
+  version = "2020";
```

Comment:
Isn't it still Haskell Report 2010? Or is this version referring to this nix build?